rotkehlchen/serialization/schemas.py

```diff
@@ -197,6 +235,7 @@ def transform_data(
         given_underlying_tokens = data.pop('underlying_tokens', None)
         chain_id = data.pop('evm_chain')
         data['chain_id'] = chain_id
+        data.pop('identifier')
```

Comment:
Perhaps it would makesense to initialize `EvmToken.initialize()` below by providing the arguments directly and getting rid of this extra logic to prepare data dict to have the exact amount of arguments required. Can you do that?